### PR TITLE
fix multicluster prometheus

### PR DESCRIPTION
**Pull request by zirain**
Please provide a description for what this PR is for.

refer to latest [prometheus config yaml](https://github.com/istio/istio/blob/b55bf1a7debc0ce0081e91ad25694e839737512b/samples/addons/prometheus.yaml#L223), we should change the demo `scrape_configs`.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [ ] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [x] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Comment by ericvn** [comment]
Waiting on approval from @istio/wg-policies-and-telemetry-maintainers

**Review by douglas-reid — APPROVED** [review]

**Review by ericvn — APPROVED** [review]

**Pull request merged**
